anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -114,9 +114,54 @@ class DataFeed(ABC):
         MapperFactory
     ] = None  # A dict/map of group names to mapper objects for translating group data into db types
 
-    def __init__(self, metadata: FeedMetadata):
+    def __init__(self, metadata: Optional[FeedMetadata] = None):
+        if not metadata:
```

Comment:
These changes make this class no longer an interface definition, but also some implementation, so either need to separate those out or update the doc string for the class to be a regular base class.

Reply:
Considering how we aren't expecting folks to extend engine with their own DataFeeds, i think making this a regular base class rather than an interface is fine.